tests/integrations/kernels/scattermoe_lora/test_ep_sentinel_skip.py

```
@@ -0,0 +1,115 @@
+    # forward equality
+    with torch.no_grad():
+        x = torch.randn(N, H, device=DEV, dtype=dt)
+        o_m = scattermoe_experts_forward(self, x, safe_idx, safe_w)
+        o_s = scattermoe_experts_forward_ep(self, x, idx, w)
+    tol = 1e-4 if dt == torch.float32 else 4e-2
+    assert torch.allclose(o_s, o_m, rtol=tol, atol=tol), (o_s - o_m).abs().max().item()
+
+    dx_m, gl_m = run(scattermoe_experts_forward, safe_idx, safe_w)
+    dx_s, gl_s = run(scattermoe_experts_forward_ep, idx, w)
+    assert torch.allclose(dx_s, dx_m, rtol=tol, atol=tol), (dx_s - dx_m).abs().max().item()
+    for a, b in zip(gl_s, gl_m):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)